samples/FunctionApp/HttpTriggerWithCancellation/HttpTriggerWithCancellation.cs

```diff
@@ -34,7 +40,7 @@ public static class HttpTriggerWithCancellation
             }
             catch (OperationCanceledException)
             {
-                logger.LogInformation("Function invocation cancelled");
+                _logger.LogInformation("Function invocation cancelled");
```

Comment:
Should we replace the `await Task.Delay(5000, cancellationToken);` with `cancellationToken.ThrowIfCancellationRequested();` ?

Reply:
Yeah I think that makes sense for the sample, I had it as a Delay for testing purposes. I wonder if I should copy the public docs here and so something like:

```csharp
if (cancellationToken.IsCancellationRequested)
{
    log.LogInformation("A cancellation token was received. Taking precautionary actions.");
    //Take precautions like noting how far along you are with processing the batch
    log.LogInformation("Precautionary activities --complete--.");
    break;
}
else
{
    //business logic as usual
    log.LogInformation($"Message: {message} was processed.");
}
```